src/component/categoryCard/CategoryCard.js

```diff
@@ -8,17 +8,17 @@ const CategoryCard = ({ value }) => {
   const { dispatch } = useData();
   return (
     <>
```

Comment:
The extra fragment is not needed here. Everything is already enclosed in the div

Reply:
Done the changes